business_rules/operators.py

```diff
@@ -276,3 +276,13 @@ def shares_exactly_one_element_with(self, other_value):
     @type_operator(FIELD_SELECT_MULTIPLE)
     def shares_no_elements_with(self, other_value):
         return not self.shares_at_least_one_element_with(other_value)
+
+
+    # This operator is only implemented for the posting rule engine at the moment
+    @type_operator(FIELD_SELECT_MULTIPLE)
+    def exists_posting_rule_engine_only(self, other_value):            
```

Comment:
can we think of a better name? how will you prevent others from using it if it's only meant to be used by posting engine folks?

Reply:
changed it to `compare_state_with_item` - my hope is that this name is only intuitive if you know exactly what it is doing and therefore will not get used accidentally